.travis.yml

```diff
@@ -6,9 +6,12 @@ php:
   - 7.2
 
 env:
-  - TYPO3_VERSION=^7.6 COVERAGE=0
-  - TYPO3_VERSION=^8.7 COVERAGE=0
-  - TYPO3_VERSION=dev-master COVERAGE=0
+  - TYPO3_VERSION=^7.6 COVERAGE=0 FUNCTIONAL_TEST=0
+  - TYPO3_VERSION=^8.7 COVERAGE=0 FUNCTIONAL_TEST=1 typo3DatabaseName=test typo3DatabaseHost=127.0.0.1 typo3DatabaseUsername=travis typo3DatabasePassword=root
```

Comment:
I would have the DB information in the command where the tests are executed, to keep the matrix cleaner.

Reply:
ok